### PR TITLE
Module 2 Capstone Project

### PR DESCRIPTION
**Pull request by PrantoshB**
### Hello there👋🏽
---

### Here are the features:
---

- [x] Homepage

         - When the page loads, the web app retrieves data from the Poké API and shows the list of items on the screen
         - The Involvement API shows the item likes

- [x] Comments Popup 
         
         - When the popup loads, the web app retrieves data from the Poké API and shows the details of the item
         - The Involvement API shows the comments 

- [x] Counters in the interfaces

         - The number of items at the homepage
         - The number of comments in the comment popup

### CODE REVIEW

File: src/modules/itemsCounter.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    const totalItems = itemsCounter('.grid-item');
+    // Assert
+    expect(totalItems).toBe(0);
+  });
+
+  test('should return 10 when there is 10 elements with "grid-item" class', () => {
+    // Arrange
+    document.body.innerHTML = '<ul class="gallery"></ul>';
+    const container = document.querySelector('.gallery');
+    for (let i = 0; i < 10; i += 1) {
```
Comment (frank1738) [inline src/modules/itemsCounter.test.js:23]: - [ ] Some of your tests are failing. Kindly fix that

image

**Review by frank1738 — COMMENTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the assessment but you are almost there!





## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/itemsCounter.test.js (hunk 1 above)
Comment (frank1738) [reply src/modules/itemsCounter.test.js:23]: Okey solved. Good work. Just remember to include the testing instructions on our readme file

Comment (PrantoshB) [reply src/modules/itemsCounter.test.js:23]: Okay @frank1738 Thanks 🤝

Comment (PrantoshB) [reply src/modules/itemsCounter.test.js:23]: @frank1738 Is there anything else we should do for the next submission? ♻️

Comment (frank1738) [reply src/modules/itemsCounter.test.js:23]: @PrantoshB. Kindly resubmit your project for approval

Comment (PrantoshB) [reply src/modules/itemsCounter.test.js:23]: Okay 😉 Thanks a lot, @frank1738.
I appreciate your efforts to reply to my questions. 🤝

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @PrantoshB , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

## Highlights 

Professional PR :+1: 
All Linter checks are passing :+1: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**